test/cmdlineTests/standard.json.stdout

```diff
@@ -1 +1 @@
-{"contracts":{"A":{"C":{"evm":{}}}},"errors":[{"component":"general","formattedMessage":"Warning: This is a pre-release compiler version, please do not use it in production.\n","message":"This is a pre-release compiler version, please do not use it in production.","severity":"warning","type":"Warning"}],"sources":{"A":{"id":0}}}
+{"contracts":{"A":{"C":{"evm":{}}}},"errors":[],"sources":{"A":{"id":0}}}
```

Comment:
That's actually weird... I have to recheck - I did run the tests locally on a release branch and they passed, but the CI run on https://circleci.com/gh/ekpyron/solidity/26 suggests that they *should* have failed, since there is indeed no empty error array...